src/Model/EventMapper.php

```diff
@@ -104,15 +110,15 @@ public function getEventById($event_id, $verbose = false, $activeEventsOnly = tr
      * @param bool $activeEventsOnly
      * @param bool $transform
      *
-     * @return false|bool the event detail
+     * @return bool|array the event detail
```

Comment:
Since this never returns `true`, `@return false|array` would be more accurate here.

Reply:
True, will check in a followup PR